.gitignore

```diff
@@ -9,3 +9,4 @@ out
 *.ipr
 target
 vagrant/.vagrant
+.settings
```

Comment:
You add `.settings` to `.gitignore`, but we already version it and you push changes to `.settings/org.eclipse.jdt.core.prefs`. We probably do not want to ignore this folder.